sunpy/util/net.py

```diff
@@ -199,3 +199,25 @@ def url_exists(url, timeout=2):
         return False
     else:
         return True
+
+
+def is_online():
+    """
+    Checks whether client is online.
```

Comment:
That's quite confusing. First I thought you  got client/server wrong... but I would say:
```
Checks whether there's an alive connection to the outside world. I.e., you are in the internet
```